src/main/scala/is/hail/variant/VariantSampleMatrix.scala

```diff
@@ -65,18 +66,18 @@ object VariantSampleMatrix {
 
   def genGeneric(hc: HailContext): Gen[VariantSampleMatrix[Annotation]] =
     for (tSig <- Type.genArb.resize(3);
-      vsm <- VSMSubgen[Annotation](
```

Comment:
I just re-loaded `code_style.xml` into IntelliJ and ran this, no change to formatting. The way you're formatting it is actually my preferred style, but it's not the style we're using. Can you export your style settings? (⌘ , -> search for "Code Style" -> next to "Scheme:", click "Manage..." -> "Export..."). Run `diff` on your file and `hail/code_style.xml`.